ember-accounts/app/templates/components/x-account.hbs

```diff
@@ -0,0 +1,89 @@
+{{#if showModal}}
+ {{#x-modal}}
+    {{component currentModal model=modalModel close='closeModal'}}
+  {{/x-modal}}
+{{/if}}
+
+<div class="alert {{if errorState 'alert-error' 'alert-success'}}">
+  <div class={{if trialing 'trialing' 'disabled'}}>
+    <p><strong> Free Trial is Active! </strong></p>
+    <p>Expires: {{stripe-date plan.trial_end}}</p>
+  </div>
+  <div class={{if trialingExpired 'trialing-expired' 'disabled'}}>
+    <p><strong> Free Trial is Expired! </strong></p>
+    <p> Upgrade your plan to enable private access </p>
+  </div>
+
+
+  <div class={{if noAccount 'new-account' 'disabled'}}>
+    <div>
+      <p>
+        <strong>All Accounts</strong> receive a <strong>15 day free trial</strong>.
+      </p>
+    </div>
+    <div class='trial-form'>
+      <form {{action "activateTrial" on="submit"}}>
+        {{input type="email" placeholder="Email Address" name="billing_email" value=emailBinding required="required"}}
+        {{input class="hb-button" type="submit" value="Activate Trial" disabled=trialButtonDisabled}}
+      </form>
+    </div>
+  </div>
+  <div class={{if active 'active' 'disabled'}}>
+    <p><strong> Subscription Active </strong></p>
+  </div>
+  <div class={{if inactive 'active' 'disabled'}}>
+    <p><strong> No Subscription Active </strong></p>
+  </div>
+  <div class={{if nonProfit 'active' 'disabled'}}>
+    <p><strong> Non-Profit Subscription </strong></p>
+  </div>
+  <div class={{if failure 'active' 'disabled'}}>
+    <p><strong> Something went wrong retrieving your account information!
+    </strong></p>
+    <p> 
+      Please try again later, or <a href="mailto:support@huboard.com">Contact Support </a>
+    </p>
+  </div>
+  <div class={{if purchaseWithTrial 'active' 'disabled'}}>
+    <p><strong> Subscription Active! </strong></p>
+    <p> 
+      As your trial is still in effect, you will not be charged until: {{stripe-date plan.trial_end}}
```

Comment:
I'm pretty sure I saw this message after upgrading a trial I had just started, but clicking back into the same org showed "Subscription Active" and a new receipt showed up. Are you able to reproduce?